cinema/views.py

```diff
@@ -12,28 +17,68 @@
     MovieDetailSerializer,
     MovieSessionDetailSerializer,
     MovieListSerializer,
+    OrderSerializer,
+    OrderListSerializer,
 )
 
 
 class GenreViewSet(viewsets.ModelViewSet):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
 
+    def list(self, request, *args, **kwargs):
+        response = super().list(request, *args, **kwargs)
+        return Response(response.data["results"])
```

Comment:
The `list` method overrides the default behavior and removes pagination metadata by returning only `response.data['results']`. Consider returning the full `response.data` to include pagination metadata.